test/markups/README.man

```diff
@@ -0,0 +1,6 @@
+.Dd $Mdocdate: March 23 1981 $
+.Dt README 1
+.Os GitHub
+.Sh README.man
+.Nm README
+.Nd This is an example readme in mandoc
```

Comment:
This is an invalid example for a second reason.  In addition to the NAME section, at least the DESCRIPTION section is also required in a manual page.  So you should probably add

    .SH DESCRIPTION
    And one line of text.

or something like that.

Reply:
Ingo probably means .Sh

Reply:
Oops, yes, of course, ".Sh DESCRIPTION", thanks for noticing.